docs/docs/specification.md

```diff
@@ -633,35 +750,78 @@ then stores it in a temporary cache if it is valid. As soon as there is at
 least a threshold of valid partial signatures, the node can aggregate them to
 create the final signature.
 
+```go
+func (v Verifier) DigestMessage(currRound uint64, prevSig []byte) []byte {
+	h := sha256.New()
+
+	if !v.scheme.DecouplePrevSig {
+		_, _ = h.Write(prevSig)
+	}
+	_, _ = h.Write(RoundToBytes(currRound))
+	return h.Sum(nil)
+}
+```
+As you can notice here, the previous signature is taken into account
+only if the scheme configuration indicates it.
+
 **Validation of beacon and storage**: Once the new beacon is created, the node
-verifies its signature, loads the last saved beacom from the database and checks
-if the following routine returns true:
+verifies its signature, loads the last saved beacon from the database and tries to 
+put it on database, applying a series of checks before it:
```

Comment:
Let's add that we dont store the previous signature in case of the unchained scheme

Reply:
Done!